Packs/CommonScripts/Scripts/GridFieldSetup/GridFieldSetup.py

```diff
@@ -0,0 +1,75 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+from typing import Dict, Any
+import traceback
+
+
+def grid_field_setup(keys: Optional[Any], vals: dict) -> List[Dict[str, Any]]:
+    """Returns a list of dictionaries based on the key/values provided.
+    :type keys: ``str``
+    :type vals: ``dict``
+    :param keys: comma separated values used for keys (columns)
+    :param vals: dictionary of the value assigned to keys
+    :return: list of dictionaries
+    """
+    if keys:
+        key_list = keys.split(',')
+    num_entries = len(key_list)
+    res_list = []
```

Comment:
why do we need this as a list?

Reply:
the structure of a grid field is a list of dictionaries